xlive/Blam/Engine/game/players.h

```diff
@@ -59,8 +68,7 @@ struct player_datum
 	int8 gap_198[2];
 	int32 field_19C;
 	int8 gap_19E[30];
-	int32 field_1BC;
-	int8 gap_1C0[60];
+	s_player_shot_info shot_info[8];
```

Comment:
add `k_shot_entries_per_player = 8` to the constants in this file and use it here